layouts/_default/baseof.html

```diff
@@ -1,25 +1,21 @@
 <html>
   <head>
-    {{ $style := resources.Get "sass/main.scss" | resources.ToCSS }}
     <title>{{ print .Title " | " .Site.Title }}</title>
-    <link rel="stylesheet" href="{{ $style.RelPermalink }}" />
-    <link rel="icon" href="/images/logo.png" />
-    <meta charset="UTF-8"> 
+    {{ partial "head" . }}
   </head>
   <body>
-    <br>
-    <div class="container">
+    <div id="main" type="page">
       {{ partial "navbar" . }}
-      <hr>
+      <div id="card">
+        <div class="card-deco">
+          <div></div><div></div><div></div><div></div>
+        </div>
```

Comment:
![image](https://github.com/osu-uwaterloo/osu-uwaterloo-website/assets/23134847/0842f4d1-1d5f-4fb1-a9b8-064116994941)
This thing. Because it has 4 polygons I cannot implement it with pseudo classes. Also I am planning to randomly generate it for different pages so that every page's deco can be different.

Reply:
oooo I see so this is one of those firefox diff moments, all good xd

thanks for the explanation